packages/admin/admin/src/stack/breadcrumbs/utils.tsx

```diff
@@ -87,3 +87,28 @@ export const useItemsToRender = (
 
     return [firstItemWithBackButton, showOverflowMenu && overflowMenu, ...remainingItems].filter((item) => item !== false);
 };
+
+export const useObservedWidth = (ref: React.RefObject<HTMLElement>): number => {
+    const [containerWidth, setContainerWidth] = React.useState(ref.current?.clientWidth ?? 0);
```

Comment:
Shouldn't this be `undefined` as initial value?

Reply:
Initially all items are rendered, regardless of the width so the initial value is not used anyway.  
Using `undefined` would require additional (unnecessary) type-checking, which is why I just chose 0.